exchange-deployer/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![no_std]
+
+multiversx_sc::imports!();
+
+pub mod action_type;
+pub mod deploy_types;
+pub mod fee;
+
+#[multiversx_sc::contract]
+pub trait ExchangeDeployer:
+    fee::FeeModule
+    + deploy_types::liquidity_pool::LiquidityPoolModule
+    + deploy_types::simple_lock::SimpleLockModule
+    + deploy_types::proxy_dex::ProxyDexModule
+    + deploy_types::farm::FarmModule
+    + deploy_types::farm_staking::FarmStakingModule
+    + deploy_types::metastaking::MetastakingModule
+    + deploy_types::common::CommonModule
+    + utils::UtilsModule
+    + multiversx_sc_modules::pause::PauseModule
+{
+    #[init]
+    fn init(
```

Comment:
Why not store all these addresses unde the same storage, but using a `DeployActionType` enum value as the key. You will remove a lot of different storages.

Reply:
Something like the `customActionFee` storage.